daml-lf/spec/daml-lf-1.rst

```diff
@@ -247,6 +243,13 @@ Version: 1.dev
 
   * **Add** ``type_rep``, a value presenting a type.
 
+Version: 1.dev
+..............
+
+  * **Change** Transaction submitter must be in the contract key
```

Comment:
When did we decide to do this? I'm not convinced requiring the submitter of a `fetchByKey` to be a maintainer of the key is a good idea. This definitely hurts the compositionality of workflows. I also don't see the technical necessity of this change. Being a stakeholder of the contract should absolutely be enough. If you know a contract with the right key exists, then it must be the unique contract with this key. If you don't see a contract with such a key, you can't submit the transaction anyway. (I'm not talking about `lookupByKey` here!)

cc @bame-da @gerolf-da

Reply:
In fact, this restriction is pretty much against the spirit of https://github.com/digital-asset/daml/issues/2311, which we all seem to have agreed on.

Reply:
This is the last work from francesco #1866 already referenced in #2311. 
I just let in dev.

Reply:
just moving it down from line 227/232 to line 246/252